check/format-incremental

```diff
@@ -1,5 +1,21 @@
 #!/usr/bin/env bash
 
+################################################################################
+# Runs flynt on the repository
+#
+# Usage:
```

Comment:
well, it is not check/flynt anymore - it is a bit confusing how this repeats the exact same lines of working directory as well. I think we can do better in merging the scripts!

Reply:
Done.